geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala

```diff
@@ -43,8 +43,11 @@ trait OptionalTypeNameParam extends TypeNameParam {
 }
 
 trait PasswordParams {
-  @Parameter(names = Array("-p", "--password"), description = "Connection password")
+  @Parameter(names = Array("-p", "--password"), description = "Connection password (will prompt if not supplied and not using Kerberos)")
   var password: String = null
+
+  @Parameter(names = Array("--keytab"), description = "Path to Kerberos keytab file")
+  var keytab: String = null
```

Comment:
@jrs53 I meant we need to move the keytab var out of this trait and into another mixin trait, and update the accumulo tools classes with the new mixin, since the PasswordParams trait is re-used in cassandra and keytabs aren't supported there (yet).